### PR TITLE
Upgrade crystal version to 1.4.1

### PR DESCRIPTION
**Pull request by alex-lairan**
Small PR to keep Crystal dependency up-to-date.

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mint
-version: 0.16.0
+version: 0.16.1
```
Comment (Sija) [inline shard.yml:2]: No need to bump version. Please revert this.

File: shard.yml [hunk 2]
```diff
@@ -30,6 +30,6 @@ targets:
   mint:
     main: src/mint.cr
 
-crystal: ~> 1.3.0
+crystal: ~> 1.4.1
```
Comment (Sija) [inline shard.yml:33]:
```suggestion
crystal: ~> 1.3
```

**Review by Sija — CHANGES_REQUESTED** [review]

File: shard.yml (hunk 2 above)
Comment (ryanprior) [reply shard.yml:33]: If you don't want to update to crystal 1.4 maybe just close the PR? I don't get why you'd request this change.

Comment (Sija) [reply shard.yml:33]: There's no need to bump minimal supported Crystal version. It has nothing to do with the _upgrading_ crystal, as it states what versions are supported, and mint support 1.3+.

**Comment by Sija** [comment]
I changed the crystal version in `shard.yml`. The entry in `.tool-versions` I'll bump once crystal 1.5.0 is out.
Thanks for the push :)

**Pull request closed without merging**